core/schema/modulesource.go

```diff
@@ -1057,21 +1057,22 @@ func callerHostFindUpContext(
 	bk *buildkit.Client,
 	curDirPath string,
 ) (string, bool, error) {
-	if !filepath.IsAbs(curDirPath) {
-		return "", false, fmt.Errorf("path is not absolute: %s", curDirPath)
-	}
+	// if !filepath.IsAbs(curDirPath) {
+	// 	return "", false, fmt.Errorf("path is not absolute: %s", curDirPath)
+	// }
```

Comment:
I think we can just remove this condition - the path should be absolute, since it comes from the client.

Reply:
If you're keeping this commented, can you add a TODO to explain?

Reply:
Woop, think we can just get rid of it.